.travis.yml

```diff
@@ -16,19 +17,45 @@ matrix:
 
 before_install:
   - mkdir -p $HOME/.local/bin
-  - export PATH=$HOME/.local/bin:$PATH
+  - export PATH=/opt/ghc/$GHCVER/bin:/opt/cabal/$CABALVER/bin:$HOME/.local/bin:$PATH
```

Comment:
It's (implicitly) from the "hvr-ghc" apt repo specified in the "matrix" section's "addons" field.